README.md

```diff
@@ -1,18 +1,21 @@
-# vRank
-Vocabulary for Ranking
+# vRank - Vocabulary for Ranking
 
 The Vocabulary for Ranking (vRank) is an RDF Schema vocabulary for materializing ranking computations.
 
 If you use this work, please mention it in your references:
 
   ```
-  @Inproceedings{j.2012tafmfsarrc,
-  booktitle = {Proceedings of the 6th International Workshop on Ranking in Databases (DBRank 2012) held in conjunction with the 38th Conference on Very Large Databases (VLDB 2012)},
-  publisher = {online},
-  title = {Towards a formal model for sharing and reusing ranking computations},
-  year = {2012},
-  type = {Inproceedings},
-  author = {Antonio J. Roa-Valverde and Andreas Thalhammer and Ioan Toma and Miguel-Angel Sicilia},
-  evastar_pdf = {Dbrank2012.pdf}
+@InProceedings{roa-valverde,
+  Title                    = {{Towards a formal model for sharing and reusing ranking computations}},
+  Author                   = {Roa-Valverde, Antonio J. and Thalhammer, Andreas and Toma, Ioan and Sicilia,Miguel-Angel},
+  Booktitle                = {Proceedings of the 6th International Workshop on Ranking in Databases (DBRank 2012) held in conjunction with the 38th Conference on Very Large Databases (VLDB 2012)},
+  Year                     = {2012},
+  Url                      = {http://km.aifb.kit.edu/services/summa/dbrank2012.pdf}
+}
   }
   ```
+
+# Changelog
+* v2.0 
```

Comment:
Should we include a little description here as well?

Reply:
That is not the latest commit - the latest includes some information